tagbot/action/repo.py

```diff
@@ -179,8 +195,15 @@ def _registry_pr(self, version: str) -> Optional[PullRequest]:
             return None
         name = self._project("name")
         uuid = self._project("uuid")
+        url = self._registry_url
+        if not url:
+            raise InvalidProject("Could not find url of package in registry")
```

Comment:
InvalidProject is usually used for bad Project.toml files, I would just log the error and then `return None` here